cmd/assets/index.gohtml

```diff
@@ -152,6 +157,10 @@
 <body>
 	<div class="container">
 		<h1>{{ .Title }}</h1>
+		<h3> Provider: {{.Provider}} </h3>
+		{{if and (eq .Provider "azure") .SpecifiedResourceGroup}}
+			<h3>Specified Resource Group: {{.SpecifiedResourceGroup}}</h3>
+		{{end}}
```

Comment:
Can we revert the changes in this file? This is more debugging information that isn't really relevant to the end user.